src/main/resources/com/mathworks/ci/RunMatlabCommandBuilder/help-matlabCommand.html

```diff
@@ -1,7 +1,8 @@
 <div>
- To run your tests, enter MATLAB commands separated by semicolons.<br>
- <b>Example:</b> cd c:\MATLAB\tests; runtests('IncludingSubfolder','true');<br>
- 
+ Enter the command you want to run in MATLAB. If the command represents a MATLAB function or script, do not specify the file extension.<br>
+ <b>Examples:</b><br>
+ <b>To run MATLAB commands :</b> cd c:\MATLAB\tests; runtests('IncludingSubfolder','true')<br> 
+ <b>To run MATLAB function or a script file :</b> cd c:\MATLAB\myscripts; runMyScript(arg1, arg2...)<br>
```

Comment:
Scripts or functions: 
cd C:\MATLAB\myMATLABCode; runMyScript; runMyFunction(arg1,arg2);

Reply:
Done.

Reply:
I am not sure why I can't see the change. However, here are my concerns:

1) How can a script accept input arguments?
2) We typically say run a function or script, but not run a "file".
3) You say "run MATLAB function or a script". The article "a" should be used consistently. (It should also appear before MATLAB)

I would rewrite the entire statement as follows:
<b>Run a function :</b> cd C:\MATLAB\myMATLABFiles; runMyFunction(arg1,arg2)<br>

Reply:
Done.